src/ui/Analytics/Analytics.ts

```diff
@@ -227,7 +227,7 @@ export class Analytics extends Component {
    *
    * > When logging custom `Search` events, you should use the `Coveo.logSearchEvent` top-level function rather than
    * > calling this method directly from the `Analytics` component instance. See
-   * > [Sending Custom Analytics Events](https://developers.coveo.com/x/KoGfAQ).
+   * > [Sending Custom Analytics Events](https://docs.coveo.com/en/365/).
```

Comment:
Link to https://docs.coveo.com/en/2726/#logging-your-own-search-events instead, and update the link name accordingly.

Reply:
The comments I made in Analytics.ts also apply to corresponding occurrences in AnalyticsClient.ts